doc/modules/clustering.rst

```diff
@@ -119,6 +121,9 @@ Gaussian mixture models, useful for clustering, are described in
 mixture models. KMeans can be seen as a special case of Gaussian mixture
 model with equal covariance per component.
 
+:term:`Transductive <transductive>` clustering methods (in contrast to :term:`inductive` clustering methods)
+are not designed to be applied to new, unseen data.
```

Comment:
```suggestion
:term:`Transductive <transductive>` clustering methods (in contrast to
:term:`inductive` clustering methods) are not designed to be applied to new,
unseen data.
```
Although it is not a very strict rule, we keep the line length <= 79 characters also in rst files.

Reply:
Forgot about that. I fixed the line lenght.